src/shared/types/index.js

```diff
@@ -87,7 +87,7 @@ export const filterType = PropTypes.shape({
     created_before: PropTypes.string,
     feedback: PropTypes.string,
     maincategory_slug: PropTypes.arrayOf(dataItemType),
-    priority: PropTypes.string,
+    priority: PropTypes.arrayOf(dataItemType),
```

Comment:
Should be `dataListType`.

Reply:
gedaan

Reply:
nee dat moet dataItemType zijn: ander krijg ik fouten: het is hetzelfde als stadsdeel, status (hieronder)

Reply:
In plaats van 
```jsx
priority: PropTypes.arrayOf(dataItemType),
```

moet het zijn:

```jsx
priority: dataListType,
```

Krijg je dan ook foutmeldingen?